ci/conda/meta.yaml

```diff
@@ -18,15 +18,15 @@ requirements:
     - ninja
 
   host:
-    - occt ==7.6
+    - occt
     - boost-cpp
     - tbb-devel
     - vtk
     - zlib
     - pthreads-win32  # [win]
 
   run:
-    - occt ==7.6
+    - {{ pin_compatible('occt', max_pin='x.x.x') }}
```

Comment:
I don't quite understand what this will do? I've always thought it was better to be more explicit with the OCCT version. Maybe this is fine I just don't know what "pin_compatible" does here.

Reply:
iirc, I think I had always specified the OCCT version because I would normally be working on pyOCCT (Python wrappers of OCCT) targeting a specific version of OCCT, and I wanted SMESH to use that same version so when I made pySMESH (Python wrappers of SMESH) they were compatible. Ultimately, I would like to use versions of OCCT and SMESH from conda-forge when distributing pyOCCT and pySMESH, so maybe it all works out in the end. It's just been a while haha...

Reply:
pin_compatible pins the version which is used in the host-env. x.x.x means that ist pins for example occt 7.7.2. Automatically (with Conda-forge-pinning) occt is not so strict pinned (x.x). There was one patch-version of occt recently which added incompatibilities, that's why we switched to x.x.x for some libraries.